docker/planet.yml

```diff
@@ -2,7 +2,7 @@ services:
   couchdb:
     expose:
       - 5984
-    image: klaemo/couchdb
+    image: couchdb:2.1.1
```

Comment:
@dogi if we want the latest of couchdb all time then we can just put couchdb on all places. Is that what you would prefer?

Reply:
I guess the `klaemo/couchdb` version is not creating the `2.1+` version and moved to `couchdb` namespace

Reply:
@mappuji you are right about the version. I was just pointing shall we do fix version or keep it as latest.